### DISCUSSION TITLE
quantumleap metadata

### DISCUSSION THREAD

**Discussion by tonyrosset** [smartsdk/ngsi-timeseries-api#764] [General]
How would I store the static metadata with Quantumleap without sacrificing my storage?

**Comment by c0c0n3** [comment]
Ha, good question. The short answer is that you can't. The QuantumLeap data model unfortunately doesn't cater for sharing the same data across entity timepoints. But you could
1. Split the time-variable (e.g. device readings) and static data (e.g. device model) into two separate entities, say T and S, respectively;
2. Keep both entities in Orion as usual;
3. Subscribe Quantum Leap to T changes but exclude S from notifications---e.g. using a pattern in the Orion subscription.
